docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/06-Zeit-Now-and-Google-Cloud-SQL.md

```diff
@@ -0,0 +1,158 @@
+
+## Prerequisites
+
+#### GCP Cloud SQL
+
+For the purpose of this tutorial you will need to have signed up for GCP, created a project and enabled the Cloud SQL Admin API (see [quickstart](https://cloud.google.com/sql/docs/mysql/) for more information).
+
+You also should install the [Cloud SDK](https://cloud.google.com/sdk/).
+
+#### Zeit Now
+
+Sign up to [zeit.co](https://zeit.co).
+
+Install the [now-cli)](https://github.com/zeit/now-cli)
+
+## 1. MySQL
+
+### 1.1. Creating A Cloud SQL Instance
+
+We'll be creating a MySQL instance called `prisma` on GCP via the command line. You may also do this via the console.
+
+```console
+❯ gcloud sql instances create prisma --tier=db-f1-micro --authorized-networks=0.0.0.0/0 --region=europe-west1
+
+NAME    DATABASE_VERSION  LOCATION        TIER         ADDRESS      STATUS
+prisma  MYSQL_5_6         europe-west1-d  db-f1-micro  xx.xx.xx.xx  RUNNABLE
+```
+
+* We are deploying to the `europe-west1` region as this is the same location as Zeit in Europe. The default region is `us-central`.
+* The network access is opened up so the database can be accessed over the internet.
+* Tier is the instance's machine type. `db-f1-micro` is the smallest possible. The default tier is `db-n1-standard-1`.
+
+---
+
+* You can learn the additional [command line flags](https://cloud.google.com/sdk/gcloud/reference/beta/sql/instances/create)
+* The steps for creating a Postgres database instance are virtually identical.
+
+
+### 1.2. Create SQL User
+
+Create a user called `prisma` with a password of `my-secret`.
+
+```console
+❯ gcloud beta sql users create prisma --instance=prisma --password=my-secret
+```
+
+Your MySQL Instance is now up and running.
+
+Note the IP address of the instance, your username and password.
+
+## 2. Deploying a Prisma server to Zeit Now
+
+Deployment to Now consists of 3 files; a Dockerfile, prisma configuration and Now deployment configuration.
+
+### 2.1 Now Secrets
+
+First we will add secrets to Now to store the sql password from earlier and the Prisma API password.
+
+```console
+❯ now secret add sql-password my-secret
+❯ now secret add prisma-management-api-secret so-secret
+```
+
+*  Learn more about Now secrets: https://zeit.co/docs/getting-started/secrets
+
+### 2.2 Prisma Configuration
+
+Create a file `config.yml`. This contains the Prisma Server configuration.
+
+```file
+managementApiSecret: PRISMA_MANAGEMENT_API_SECRET
+port: 4466
+databases:
+    default:
+        connector: mysql
+        host: SQL_HOST
+        port: 3306
+        user: prisma
+        password: SQL_PASSWORD
+        migrations: true
+        active: true
+```
+
+* SQL_HOST, SQL_PASSWORD and PRISMA_MANAGEMENT_API_SECRET will be replaced with Now build-time environment variables.
```

Comment:
surround `SQL_HOST`, `SQL_PASSWORD` and `PRISMA_MANAGEMENT_API_SECRET` with backticks so they're formatted as inline code